app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -1011,16 +1011,17 @@ def self.snapshot_inv_to_snapshot_hashes(inv, current, parent_uid = nil)
         result
       end
 
-      def self.folder_and_dc_inv_to_hashes(folder_inv, dc_inv)
-        folder_result, folder_uids = folder_inv_to_hashes(folder_inv, false)
-        dc_result, dc_uids = folder_inv_to_hashes(dc_inv, true)
+      def self.folder_dc_sp_inv_to_hashes(folder_inv, dc_inv, sp_inv)
```

Comment:
I know this method name would be long but I'm not sure I would remember dc and sp are datacenter and storage pod.